src/main/scala/ch/epfl/bluebrain/nexus/kg/resolve/Resolver.scala

```diff
@@ -45,6 +53,23 @@ sealed trait Resolver extends Product with Serializable {
     * @return the resolver priority
     */
   def priority: Int
+
+  /**
+    * Converts a resolver to a [[ResourceF.Value]]
+    *
+    * @param id            the id of the resource
+    * @param flattenCtxObj the flatten context object (not wrapped in the @context key)
+    */
+  def toResourceV(id: ResId, flattenCtxObj: Json): ResourceF.Value = {
```

Comment:
rename method to `resourceValue` as `ResourceV` is really a `ResourceF` parameterized on the `Value` type (it contains a `Value`)
... or return a `ResourceV` if there's enough information